t/lib/t/MusicBrainz/Server/Controller/Artist/Aliases.pm

```diff
@@ -59,7 +54,6 @@ $mech->content_unlike(qr/Test Alias/, 'other artist pages do not have the alias'
 page_test_jsonld $mech => {
     '@context' => 'http://schema.org',
     '@type' => 'MusicGroup',
-    'alternateName' => [],
```

Comment:
We don't return empty lists for anything else, so I assume this was just an oversight.